src/LaminasModuleProvider.php

```diff
@@ -179,7 +181,7 @@ public function getHydratorConfig() : array
         return $this->convert($this->module->getHydratorConfig());
     }
 
-    public function getInputFilterConfig() /* : array */
+    public function getInputFilterConfig() /* phpcs:ignore */
```

Comment:
I know CS checks are ignored, but we can still document the types in the docblock, via `/** @return array */` :) 

Same applies for the other `phpcs:ignore` locations

Reply:
Obviously. Wtf. My brain is out of order.

Reply:
Nah, it's all working as expected, it's just that every project ticks differently :D